src/TourManager/Travely.TourManager.DAL/Attachment.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Travely.TourManager.DAL
+{
+    [Table("Attachment")]
+    public class Attachment
+    {
+        [Key]
+        [Column("Id")]
```

Comment:
Move configurations logic to `IEntityTypeConfiguration` or to `OnModelCreating`.